[2025-10-06T08:25:33.862+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: inventory_analytics_pipeline.create_inventory_table manual__2025-10-06T08:25:23.670492+00:00 [queued]>
[2025-10-06T08:25:33.891+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: inventory_analytics_pipeline.create_inventory_table manual__2025-10-06T08:25:23.670492+00:00 [queued]>
[2025-10-06T08:25:33.892+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-10-06T08:25:33.931+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): create_inventory_table> on 2025-10-06 08:25:23.670492+00:00
[2025-10-06T08:25:33.960+0000] {standard_task_runner.py:60} INFO - Started process 628 to run task
[2025-10-06T08:25:34.002+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'inventory_analytics_pipeline', 'create_inventory_table', 'manual__2025-10-06T08:25:23.670492+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/inventory_analytics_pipeline.py', '--cfg-path', '/tmp/tmpugwqvkqp']
[2025-10-06T08:25:34.102+0000] {standard_task_runner.py:88} INFO - Job 14: Subtask create_inventory_table
[2025-10-06T08:25:35.105+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-10-06T08:25:35.664+0000] {task_command.py:423} INFO - Running <TaskInstance: inventory_analytics_pipeline.create_inventory_table manual__2025-10-06T08:25:23.670492+00:00 [running]> on host ea4788e59d79
[2025-10-06T08:25:36.664+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='inventory_analytics_pipeline' AIRFLOW_CTX_TASK_ID='create_inventory_table' AIRFLOW_CTX_EXECUTION_DATE='2025-10-06T08:25:23.670492+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-06T08:25:23.670492+00:00'
[2025-10-06T08:25:36.771+0000] {base.py:83} INFO - Using connection ID 'postgres_default' for task execution.
[2025-10-06T08:25:36.804+0000] {sql.py:450} INFO - Running statement: 
    CREATE TABLE IF NOT EXISTS inventory (
        id SERIAL PRIMARY KEY,
        product_name VARCHAR(100),
        quantity INTEGER,
        warehouse VARCHAR(50)
    );
    , parameters: None
[2025-10-06T08:25:37.129+0000] {base.py:83} INFO - Using connection ID 'postgres_default' for task execution.
[2025-10-06T08:25:37.147+0000] {sql.py:450} INFO - Running statement: DELETE FROM inventory;, parameters: None
[2025-10-06T08:25:37.153+0000] {sql.py:459} INFO - Rows affected: 0
[2025-10-06T08:25:37.185+0000] {base.py:83} INFO - Using connection ID 'postgres_default' for task execution.
[2025-10-06T08:25:37.239+0000] {sql.py:450} INFO - Running statement: 
    INSERT INTO inventory (product_name, quantity, warehouse)
    VALUES
        ('Laptop', 10, 'Warehouse A'),
        ('Mouse', 50, 'Warehouse B'),
        ('Keyboard', 30, 'Warehouse A'),
        ('Monitor', 15, 'Warehouse B');
    , parameters: None
[2025-10-06T08:25:37.246+0000] {sql.py:459} INFO - Rows affected: 4
[2025-10-06T08:25:37.254+0000] {logging_mixin.py:188} INFO - âœ… Inventory table created and sample data inserted successfully
[2025-10-06T08:25:37.257+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-10-06T08:25:37.388+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=inventory_analytics_pipeline, task_id=create_inventory_table, execution_date=20251006T082523, start_date=20251006T082533, end_date=20251006T082537
[2025-10-06T08:25:38.476+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-10-06T08:25:38.569+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
